src/classes/HelloPortland.cls

```diff
@@ -1,11 +1,12 @@
 public with sharing class HelloPortland {
 	public static String sayHello(String name) {
 		return String.format('Hello {0}!', new List<String> {name});
+		// comment and cool stuff here.
 	}
 
 	public static void badThing(Integer i) {
 		for (Integer j=0; j<i; j++) {
-			insert new Account(Name = 'New Account ' + i);
+			insert new Account(Name = 'New Account for fun ' + i);
```

Comment:
This DML statement is inside a loop and is highly likely to hit governor limits. Please use collections to move the DML statement outside of the loop.